frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "react-dom": "^18.2.0",
     "react-router": "^6.3.0",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
```

Comment:
why has this been downgraded?

Reply:
Tbh not sure. This was all either already in there or automatically done.

Reply:
i appreciate your honesty